code/datums/diseases/_disease.dm

```diff
@@ -155,7 +155,7 @@ var/list/diseases = subtypesof(/datum/disease)
 		if(disease_flags & CAN_RESIST)
 			if(!(type in affected_mob.resistances))
 				affected_mob.resistances += type
-				remove_virus()
+		remove_virus()
```

Comment:
I didn't intend to add this to the PR, but whatever.
I'm not sure why we wouldn't want to remove the virus when we have a resistance to it?